libs/s25main/desktops/dskCredits.cpp

```diff
@@ -322,7 +453,20 @@ glArchivItem_Bitmap* dskCredits::GetCreditsImgOrDefault(const std::string& name)
 
 bool dskCredits::Msg_KeyDown(const KeyEvent& /*ke*/)
 {
-    return Close();
+    GotoNextPage();
+    return true;
+}
+
+bool dskCredits::Msg_LeftUp(const MouseCoords& /*mc*/)
+{
+    GotoNextPage();
+    return true;
+}
+
+bool dskCredits::Msg_RightUp(const MouseCoords& mc)
```

Comment:
```suggestion
bool dskCredits::Msg_RightUp(const MouseCoords&)
```
or the warning will trigger